lib/inventory_refresh/inventory_collection.rb

```diff
@@ -597,8 +593,6 @@ def new_inventory_object(hash)
       inventory_object_class.new(self, hash)
     end
 
-    attr_writer :attributes_blacklist, :attributes_whitelist
-
```

Comment:
Oh here's the write bit...how strange.

Reply:
Yeah unclear why this was way down away from the others, but not private